### ISSUE TITLE
clarify Documentation

### ISSUE THREAD

**Issue by knbknb** [petli-full/jq-vscode#1]
Thanks for the brilliant extension. I managed to get it to work on my Linux PC. 

Please clarify the documenation a little bit.

At this time your screenshots (the animated GIFs) show two files which are both named "Untitled" . This is kind of confusing, because the text says something like 'Open a .json file, and then open a .jqx file'.  

However, your extension seems to try to apply jq-commands to *any* file that is currently in focus (where the tab is selected/highlighted), and then there must always be a new empty file  opened with CTRL+SHIFT+P to activate the jq-vscode extension. Only then can jq expressions be applied on the previously highlighted tab. The jq-expression file is always named 'Untitled', not `[something].jqx`. And I was not able to save it with such an etension. This breaks the  sync with the JsoN file-tab

So, The "Open .jqx file" seems unnecessary. 

Besides, you should write what a .jqx file is. What are they used for? 

JQX Format seems to be your own invention. On the jq manual page there is no mention of `.jqx`. A Google search for  files with .jqx extension says: "What is a JQX file? The JQX file type is primarily associated with Windows Help Related File by Microsoft Corporation. Typically seen in the form WIN_QS."

**Comment by petli-full** [comment]
thanks for your feedback! I've updated the doc a bit. Hope it's less confusing this time. 

The extension requires to split the active editor into two: one for json, and the other for jq. As you've probably already noticed, the jqx is used by the extension itself to tell which doc currently contains jq. There's no need for users to see the detail. The unnecessary text was removed. The screenshots were also updated for more demos.
